kalite/topic_tools/__init__.py

```diff
@@ -515,6 +414,9 @@ def get_content_data(request, content_id=None):
         elif not request.is_logged_in:
             messages.warning(request, _("This content was not found! You must login as an admin/coach to download the content."))
 
+    annotations = content.annotations()
+    content.update(annotations)
```

Comment:
With these lines `get_content_data` takes about 1.7s/execution, and without it's still about 0.7s/execution. I counted about 7377 content items, so reducing execution time to about 1ms would put us in the 7s range for `get_topic_tree`.